content/queues/changelog.md

```diff
@@ -11,3 +11,7 @@ weight: 11
 ### Increased per-account limits
```

Comment:
Should we include a second-level header for the new change like Matt did for this one?

Reply:
Good call.